.metadocrc

```diff
@@ -0,0 +1,15 @@
+{
+	"files": [
+	    "impress.js",
+	    "lib/client.js"
+	],
+	"customLinks": {
+	    "Application": "https://github.com/metarhia/impress/blob/master/lib/application.js",
+	    "IncomingMessage": "https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_class_http_incomingmessage",
```

Comment:
Same as below:
```suggestion
	    "IncomingMessage": "https://nodejs.org/api/http.html#http_class_http_incomingmessage",
```

Reply:
Also, I think it should be named `http.IncomingMessage`, not just `IncomingMessage`
```suggestion
	    "http.IncomingMessage": "https://nodejs.org/dist/latest-v10.x/docs/api/http.html#http_class_http_incomingmessage",
```